mlrun/config.py

```diff
@@ -937,6 +940,79 @@ def get_v3io_access_key(self):
         # Get v3io access key from the environment
         return os.environ.get("V3IO_ACCESS_KEY")
 
+    def get_file_target_path(
```

Comment:
```suggestion
    def get_model_monitoring_file_target_path(
```
This is specific to model monitoring, should reflect that.